Scripts/install-prerequisites.cmd

```diff
@@ -0,0 +1,250 @@
+@echo off
```

Comment:
Please remove this file and the processes that call it..

Reply:
Removed files